datamodel/testdata.d/07_test_data_tnt.sql

```diff
@@ -149,7 +155,7 @@ INSERT INTO dcsa_im_v3_0.equipment_event (
     'EQUIPMENT',
     TO_DATE('2003/05/03 21:02:44', 'yyyy/mm/dd hh24:mi:ss'),
     'ARRI',
-    uuid('8b64d20b-523b-4491-b2e5-32cfa5174eed'),
+    '8b64d20b-523b-4491-b2e5-32cfa5174eed',
```

Comment:
You are changing here but not in transport_event below. I suspect there are many more places where we insert UUIDs which now should be string

Reply:
Either we change all palces or none - we don't change 1 place and not the rest...